Snippets/Core/Core_8/UpgradeGuides/DisablePublishing.cs

```diff
@@ -1,28 +1,43 @@
 ﻿namespace Core8.PublishSubscribe
 {
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
     using NServiceBus;
     using NServiceBus.Routing;
-    using NServiceBus.Settings;
     using NServiceBus.Transport;
 
     class DisablePublishingUpgradeGuide
     {
         void DisablePublishingConfiguration(EndpointConfiguration endpointConfiguration)
         {
             #region DisablePublishing-UpgradeGuide
-            var transportConfiguration = endpointConfiguration.UseTransport<TransportDefinition>();
+            var transportConfiguration = endpointConfiguration.UseTransport(new TransportDefinition());
             transportConfiguration.DisablePublishing();
             #endregion
         }
 
         class TransportDefinition : NServiceBus.Transport.TransportDefinition, IMessageDrivenSubscriptionTransport
```

Comment:
given the amount of times a custom transport is being used for snippets, it might be easier to just define this once and reuse across all snippets?